arangod/Aql/OptimizerRulesFeature.cpp

```diff
@@ -245,6 +245,8 @@ void OptimizerRulesFeature::addRules() {
   registerRule("patch-update-statements", patchUpdateStatementsRule,
                OptimizerRule::patchUpdateStatementsRule_pass9, DoesNotCreateAdditionalPlans, CanBeDisabled);
 
+  registerRule("replace-near-within-fulltex", replaceNearWithinFulltext,
```

Comment:
If I understand correctly, we are replacing an AQL function with the usage of an index node. That's what the user will care about, and that's what logically groups these three functions. So maybe `registerRule("replace-function-with-index", replaceFunctionWithIndex,...`. Right?